src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/TeleporterSystem.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.ligthandshadow.componentsystem.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterMode;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.ligthandshadow.componentsystem.components.LASTeam;
+import org.terasology.ligthandshadow.componentsystem.components.SetTeamOnActivateComponent;
+import org.terasology.logic.characters.CharacterTeleportEvent;
+import org.terasology.logic.common.ActivateEvent;
+import org.terasology.math.geom.Vector3f;
+
+@RegisterSystem(RegisterMode.AUTHORITY)
+public class TeleporterSystem extends BaseComponentSystem {
+
+    private static final Logger LOG = LoggerFactory.getLogger(TeleporterSystem.class);
```

Comment:
If you take a look at the codebase, everywhere the Logger is named logger. I would request that you stick to that name, for consistency.

Reply:
(also, read https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case)